src/App.tsx

```diff
@@ -178,6 +184,24 @@ const Content = () => {
   );
 };
 
+function useUiRefreshMigrations() {
+  const themeSetting = useAppSelector(getAppThemeSetting);
+  const dispatch = useAppDispatch();
+  const { uiRefresh } = testFlags;
+  const [seenUiRefresh, setSeenUiRefresh] = useLocalStorage({
+    key: LocalStorageKey.HasSeenUiRefresh,
+    defaultValue: false,
+  });
+  useEffect(() => {
+    if (uiRefresh && !seenUiRefresh) {
+      setSeenUiRefresh(true);
+      if (themeSetting === AppTheme.Classic) {
+        dispatch(setAppThemeSetting(AppTheme.Dark));
+      }
+    }
```

Comment:
this almost feels like we could do within a redux-persist migration

Reply:
and can add analytics within the migration file as well

Reply:
we could but we'd have to push that migration in the same commit/release as the pr where we turn on the uiRefresh flag, otherwise it does nothing one time and it's done.